doc/api_rstgen.py

```diff
@@ -144,7 +144,7 @@ def _write_common_rst_members(rst_file):
 
 def _generate_api_source_rst_files(folder: str, files: list):
     for file in files:
-        if "index" in file:
+        if "contents" in file:
```

Comment:
Is the condition that `file` ends with "_contents"?

Reply:
Yes, we can use that condition as well, updated. Thank you.